addons/l10n_id/__manifest__.py

```diff
@@ -13,12 +13,13 @@
     'author': 'vitraining.com',
     'website': 'https://www.odoo.com/documentation/master/applications/finance/fiscal_localizations/indonesia.html',
     'depends': [
-        'account',
+        'account_qr_code_emv',
```

Comment:
I dont think you should depends on emv bcs you are not using emv qr code format.

Reply:
in that case, can we move 'country_code' field of res.partner.bank from env to `base` or  `account`?

Reply:
https://github.com/odoo/odoo/blob/910756df630c52ac143a4d25a4acd06bf2131cd5/addons/account_qr_code_emv/models/res_bank.py

Reply:
I think its time to move it. What do you think @vin-odoo?

Reply:
Yeah it's just a related, go for it and move this into the account module if it's needed :+1: 
@nni-odoo @tong-odoo

Reply:
very cool thanks! @vin-odoo @tong-odoo